app.rb

```diff
@@ -0,0 +1,99 @@
+require './person'
+require './classroom'
+require './book'
+require './student'
+require './rental'
+require './teacher'
+
+class App
+  def initialize
+    @classroom = Classroom.new('Unknown')
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "#{person.role} - #{person.name}" }
+  end
+
+  def create_student(age, name, permission)
+    @people << Student.new(@classroom, age, name, permission) # Use @classroom instead of classroom:
+    puts "Student #{name} created successfully!"
+  end
+
+  def create_teacher(age, name, specialization)
+    @people << Teacher.new(specialization, age, name)
+    puts "Teacher #{name} created successfully!"
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]'
+    user_input3 = gets.chomp
+
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    if user_input3 == '1'
+      print 'Has parent permission? [Y/N]: '
+      permission = gets.chomp
+      permission = permission.downcase == 'y'
+      create_student(age, name, permission)
+    elsif user_input3 == '2'
+      print 'Specialization: '
+      specialization = gets.chomp
+      create_teacher(age, name, specialization)
+    else
+      puts 'Invalid Option'
+    end
+  end
```

Comment:
- [ ] Kindly make sure there are no errors when your app is run in the terminal if I click to create a teacher I encounter an error, see screenshot below 👍🏿 
![Screenshot_121](https://github.com/Mar12358/school-library-people/assets/69638013/370d218b-1e57-4056-a776-6e79b9a6f272)
![Screenshot_120](https://github.com/Mar12358/school-library-people/assets/69638013/8baac658-0376-46d4-9ba3-7d35fc867a9b)

Reply:
Thanks, but I already know, that's why I documented the issue on the PR and asked for feedback.
When I solve this, a Linter error appears. I also have an issue with `person.role`, when trying to list all people, could you please give me thorough feedback on both issues?
![image](https://github.com/Mar12358/school-library-people/assets/120040536/58defacf-ca92-491d-8b75-c3dfc477909d)